irispy/iris_tools.py

```diff
@@ -802,7 +802,7 @@ def uncalculate_exposure_time_correction(old_data_arrays, old_unit,
 
 
 def convert_or_undo_photons_per_sec_to_radiance(data_quantities,
-        response_version, obs_wavelength, detector_type,
+        time_obs, obs_wavelength, detector_type,
```

Comment:
```response_version``` is still required (see [here](https://github.com/sunpy/irispy/pull/117/files/e49ae96b2b60bd5b7a56374a57d7befac1724497..cdd3b20cf1386fc3bffafc033a0e094ae467a271#diff-dee23d5475a6ed5e368938455c92fb2eR862)) and so should be added back in while still keeping ```time_obs```.